packages/docusaurus-theme-classic/src/theme/DocPage/styles.module.css

```diff
@@ -19,10 +19,6 @@
   width: 100%;
 }
 
-.docItemWrapper {
-  padding-top: 0.5rem !important;
```

Comment:
Now top padding will be slightly larger instead of 0.5 -> 1rem, but it's still good.